doxydocs/man/man3/parts_speech.3
.TH "parts_speech" 3 "Mon Dec 12 2022" "natural english language tokenizer" \" -*- nroff -*-
.ad l
.nh
.SH NAME
parts_speech
.SH SYNOPSIS
.br
.PP
.SS "Variables"

.in +1c
.ti -1c
.RI "list \fBnouns\fP"
.br
.ti -1c
.RI "list \fBverbs\fP"
.br
.ti -1c
.RI "list \fBadjectives\fP"
.br
.ti -1c
.RI "list \fBadverbs\fP"
.br
.ti -1c
.RI "list \fBconjunctions\fP"
.br
.ti -1c
.RI "list \fBinterjections\fP"
.br
.ti -1c
.RI "list \fBpronouns\fP"
.br
.in -1c
.SH "Variable Documentation"
.PP 
.SS "list parts_speech\&.adjectives"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "bad", "backup", "bacteria", "baffling", "balanced", "ballsy", "balmy", "balsamic", "banging", "baronial", "bashful", "beady", "beaming", "bearing", "beatific",
3     "beautified", "beautifu", "beefy", "beguiling", "beginning", "bejeweled", "believable", "belligerent", "bell-like", "beloved", "benedictory", "benefic", "beneficent",
4     "beneficiary", "benevolent", "benign", "benignant", "bent", "best", "better", "bewitching", "becameral", "big", "biggest", "big-hearted", "big-time", "bijou",
5     "blameless", "blazing", "blessed", "blest", "blissful", "blistering", "blithe", "blooming", "blue-ribbon", "blushing", "bodacious", "boisterous", "bold", "bomb",
6     "bombastic", "bonkers", "bonny", "bonzer", "bookish", "boon", "bootylicious", "bosom", "botanical", "bouncy", "bound", "boundless", "bounteous", "bountiful", "bovine",
7     "bracing", "brackish", "brainy", "brash", "brave", "brawny", "brazen", "breathtaking", "breezy", "brief", "bright", "brill", "brilliant", "brimming", "brisk",
8 ]
.fi
.PP
Definition at line \fB20\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.adverbs"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "badly", "back", "backstage", "backward", "badly", "barely", "basically", "beautifully", "beforehand", "begrudgingly", "behaviorally", "balatedly", "beliievably",
3     "below", "beneficlly", "bi-monthly", "biblically", "biennially", "bilaterally", "bimonthly", "biologically", "bit", "bitterly", "biweekly", "bizarrely", "blandly",
4     "blanky", "blatanly", "blazingly", "blessedly", "blindingly", "blindly", "blissfully", "blisteringly", "blithely", "bluntly", "boldly", "boringly", "botanically",
5     "bravely", "brazenly", "breathlessly", "breathtakingly", "briefly", "brightly", "brilliantly", "briskly", "broadly", "brutally", "busily", "braggingly", "bragly", "brashly",
6 ],
.fi
.PP
Definition at line \fB30\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.conjunctions"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "because", "before", "behind", "below", "beneath", "beside", "between", "beyond", "but", "by", "by means of", "by virtue of", "by way of", "by virtue of", "by way of"]
.fi
.PP
Definition at line \fB38\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.interjections"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "bah", "bam", "bang", "barf", "bark", "bawl", "beep", "behold", "bellow", "bend", "beware"]
.fi
.PP
Definition at line \fB42\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.nouns"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "bachelor", "backbone", "balance", "brightness", "benefit", "being", "best", "booster", "bachelor", "babe", "black", "board",
3     "boarder", "broker", "bestseller", "beverage", "bean", "bail", "beast", "batch", "behind", "bracket", "brace", "builder", "beefalo",
4     "beef", "battery", "barn", "bone", "back", "butter", "bread", "beeper", "bowels", "bacon", "blaze", "blowup", "blackener", "blackhead",
5     "brisket", "barberry", "blueberries", "bus stop", "brow", "bale", "birthplace", "berlin", "bubbles", "buffer", "bucket", "bleachers",
6     "bike", "book", "bar", "breeze", "best", "backing", "block", "blackpool", "border", "bracelet", "bobbery", "bunkup", "band", "bucking",
7     "bag", "bootes", "bowtie", "bridle", "bettery", "backhoe", "bluebill", "barnacle", "bongo", "beef", "bicycle", "bulldog", "bluebill",
8 ]
.fi
.PP
Definition at line \fB2\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.pronouns"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "both", "but", "by", "by means of", "by virtue of", "by way of", "by virtue of", "by way of"]
.fi
.PP
Definition at line \fB45\fP of file \fBparts_speech\&.py\fP\&.
.SS "list parts_speech\&.verbs"
\fBInitial value:\fP
.PP
.nf
1 =  [
2     "bake", "bark", "back", "backfire", "balance", "band", "bank", "baptize", "bar", "bargain", "bark", "barrack", "barter", "base", "bash", "bask",
3     "baste", "bat", "barrow", "bash", "bathe", "battle", "bawl", "be", "beach", "bear", "beat", "beath", "beatify", "beblood", "bebleed", "beblot", "beclap",
4     "becurl", "bedas", "bedribble", "bedrop", "bedrug", "beduck", "been", "bedflatter", "bedflower", "befrill", "beg", "begin", "begird", "beg",
5     "behead", "behoof", "being", "bejumble", "belate", "belk", "belong", "belt", "bemeet", "bemoil", "bemuse", "bename", "bend", "benefits", "bench",
6     "beray", "bereave", "berime", "bescreen", "beseek", "beshroud", "besit", "beslave", "beslime", "besnow", "besort", "bespice", "best", "bestain",
7     "betake", "betitle", "bethrall", "betoss", "betrap", "better", "bevel", "beware", "bewitch", "bewrap", "bicycle", "bike", "bill", "bind", "birth"]
.fi
.PP
Definition at line \fB12\fP of file \fBparts_speech\&.py\fP\&.
.SH "Author"
.PP 
Generated automatically by Doxygen for natural english language tokenizer from the source code\&.


parts_speech.py
# list of nouns starting with b
nouns = [
    "bachelor", "backbone", "balance", "brightness", "benefit", "being", "best", "booster", "bachelor", "babe", "black", "board",
    "boarder", "broker", "bestseller", "beverage", "bean", "bail", "beast", "batch", "behind", "bracket", "brace", "builder", "beefalo",
    "beef", "battery", "barn", "bone", "back", "butter", "bread", "beeper", "bowels", "bacon", "blaze", "blowup", "blackener", "blackhead",
    "brisket", "barberry", "blueberries", "bus stop", "brow", "bale", "birthplace", "berlin", "bubbles", "buffer", "bucket", "bleachers",
    "bike", "book", "bar", "breeze", "best", "backing", "block", "blackpool", "border", "bracelet", "bobbery", "bunkup", "band", "bucking",
    "bag", "bootes", "bowtie", "bridle", "bettery", "backhoe", "bluebill", "barnacle", "bongo", "beef", "bicycle", "bulldog", "bluebill",
]

# list of verbs starting with the letter "b"
verbs = [
    "bake", "bark", "back", "backfire", "balance", "band", "bank", "baptize", "bar", "bargain", "bark", "barrack", "barter", "base", "bash", "bask",
    "baste", "bat", "barrow", "bash", "bathe", "battle", "bawl", "be", "beach", "bear", "beat", "beath", "beatify", "beblood", "bebleed", "beblot", "beclap",
    "becurl", "bedas", "bedribble", "bedrop", "bedrug", "beduck", "been", "bedflatter", "bedflower", "befrill", "beg", "begin", "begird", "beg",
    "behead", "behoof", "being", "bejumble", "belate", "belk", "belong", "belt", "bemeet", "bemoil", "bemuse", "bename", "bend", "benefits", "bench",
    "beray", "bereave", "berime", "bescreen", "beseek", "beshroud", "besit", "beslave", "beslime", "besnow", "besort", "bespice", "best", "bestain",
    "betake", "betitle", "bethrall", "betoss", "betrap", "better", "bevel", "beware", "bewitch", "bewrap", "bicycle", "bike", "bill", "bind", "birth"]
# list of adjectives starting with the letter "b"
adjectives = [
    "bad", "backup", "bacteria", "baffling", "balanced", "ballsy", "balmy", "balsamic", "banging", "baronial", "bashful", "beady", "beaming", "bearing", "beatific",
    "beautified", "beautifu", "beefy", "beguiling", "beginning", "bejeweled", "believable", "belligerent", "bell-like", "beloved", "benedictory", "benefic", "beneficent",
    "beneficiary", "benevolent", "benign", "benignant", "bent", "best", "better", "bewitching", "becameral", "big", "biggest", "big-hearted", "big-time", "bijou",
    "blameless", "blazing", "blessed", "blest", "blissful", "blistering", "blithe", "blooming", "blue-ribbon", "blushing", "bodacious", "boisterous", "bold", "bomb",
    "bombastic", "bonkers", "bonny", "bonzer", "bookish", "boon", "bootylicious", "bosom", "botanical", "bouncy", "bound", "boundless", "bounteous", "bountiful", "bovine",
    "bracing", "brackish", "brainy", "brash", "brave", "brawny", "brazen", "breathtaking", "breezy", "brief", "bright", "brill", "brilliant", "brimming", "brisk",
]
# list of adjectives starting with letter "b"
# list of adverbs starting with the letter "b"
adverbs = [
    "badly", "back", "backstage", "backward", "badly", "barely", "basically", "beautifully", "beforehand", "begrudgingly", "behaviorally", "balatedly", "beliievably",
    "below", "beneficlly", "bi-monthly", "biblically", "biennially", "bilaterally", "bimonthly", "biologically", "bit", "bitterly", "biweekly", "bizarrely", "blandly",
    "blanky", "blatanly", "blazingly", "blessedly", "blindingly", "blindly", "blissfully", "blisteringly", "blithely", "bluntly", "boldly", "boringly", "botanically",
    "bravely", "brazenly", "breathlessly", "breathtakingly", "briefly", "brightly", "brilliantly", "briskly", "broadly", "brutally", "busily", "braggingly", "bragly", "brashly",
],

# conjunctions starting with the letter "b"
conjunctions = [
    "because", "before", "behind", "below", "beneath", "beside", "between", "beyond", "but", "by", "by means of", "by virtue of", "by way of", "by virtue of", "by way of"]

# list of intergecting conjunctions starting with the letter "b"
interjections = [
    "bah", "bam", "bang", "barf", "bark", "bawl", "beep", "behold", "bellow", "bend", "beware"]
# list of pronouns starting with the letter "b"
pronouns = [
    "both", "but", "by", "by means of", "by virtue of", "by way of", "by virtue of", "by way of"]